frontend/src/pages/ProcessDataPage/ProcessedDataPage.tsx

```diff
@@ -73,7 +76,11 @@ const ProcessedDataPage = function () {
         if (currentFile) {
             const { id, watch } = currentFile;
             const lowerCaseWatch = watch.toLowerCase();
+
+            // TODO: Do a better job of guestimating percentages...
+            setPercentage(20);
             handlePredict(id, selectedModel, lowerCaseWatch);
```

Comment:
Make sure this needs to be await to make sure this line finishes before the percentage finishes
```suggestion
            await handlePredict(id, selectedModel, lowerCaseWatch);
```

Reply:
That makes sense. Thanks!